__tests__/app.test.js

```diff
@@ -324,6 +324,44 @@ describe('POST /api/articles/:article_id/comments', () => {
     })
 
 });
+describe('add query functionality GET /api/articles ', () => {
+    test('Should respond with a an appropriately sorted array of articles', () => {
+        return request(app)
+        .get('/api/articles?sort_by=created_at&order=ASC')
+        .expect(200)
+        .then(({body: {articles}})=> {
+            expect(articles).toBeSortedBy("created_at", {
+                ascending: true,
+            });
+        });
+    });
```

Comment:
It would be good to see a separate test demonstrating the sort_by, at the moment this test is using the default